app/validators/blocks/block_validator.py

```diff
@@ -7,6 +7,9 @@ class BlockValidator(Validator):
     ANSWER_REFERENCE_INVALID = "Invalid answer reference"
     LIST_REFERENCE_INVALID = "Invalid list reference"
     ANSWER_SELF_REFERENCE = "Invalid answer reference (self-reference)"
+    LIST_NAME_MISSING = "List name defined in action params does not exist"
+    BLOCK_ID_MISSING = "Block id defined in action params does not exist"
+    ACTION_PARAMS_MISSING = "Action params does not exist"
```

Comment:
This message should be consistent with `LIST_COLLECTOR_KEY_MISSING`, something like:
```suggestion
    ACTION_PARAMS_MISSING = "Action params key missing"
```